app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -126,4 +126,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   def unauthenticatedFeedbackUrl(): String =
     s"$feedbackUnauthenticatedLink?service=${serviceIdentifier}"
 
+  lazy val isLiabilityPreLaunchEnabled: Boolean =
```

Comment:
I think `isPreLaunch` is more flexible and easier to read.

Reply:
Addressed